mkdocs/themes/readthedocs/mkdocs_theme.yml

```diff
@@ -13,10 +13,16 @@ hljs_languages: []
 
 analytics:
   gtag: null
+  # Set to true to anonymize the ip for Google Analytics
+  anonymize_ip: false
 
 include_homepage_in_sidebar: true
 prev_next_buttons_location: bottom
 navigation_depth: 4
 titles_only: false
 sticky_navigation: true
 collapse_navigation: true
+# Should be the css background shorthand, using single quotes.
+style_nav_header_background:
```

Comment:
I'd much prefer this `style_nav_header_background` to be dropped. MkDocs already makes it customary to add CSS to a separate file, and people can just write `.wy-side-nav-search { whatever }`

Reply:
I agree. I'll remove it.